assets/training/model_management/src/azureml/model/mgmt/processors/pyfunc/stable_diffusion/mlflow_convertor.py

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+import mlflow
+import os
+from pathlib import Path
+from typing import Dict
+
+from .mlflow_wrapper import StableDiffusionMLflowWrapper
```

Comment:
I think, relative path don't work with mlflow code. We generally add path in system path. Please verify it if it's working.